src/components/Header/Header.jsx

```diff
@@ -63,10 +67,34 @@ function Header() {
             alignItems: "center" /* Ensure they are vertically aligned */
 
          }}>
+          
+
+<footer class="bg-white rounded-lg shadow dark:bg-gray-900 m-4">
+    <div class="w-full max-w-screen-xl mx-auto p-4 md:py-8">
+        <div class="sm:flex sm:items-center sm:justify-between">
+
           <a href='https://github.com/manzil-infinity180' style={{textDecoration:"none",margin: "10px"}}><h1 className='h1_header'>About Me</h1></a>
```

Comment:
here too

Reply:
changes are done please review and merge it @manzil-infinity180